src/solvers/flattening/bv_minimize.cpp

```diff
@@ -56,8 +56,7 @@ void bv_minimizet::operator()(const minimization_listt &symbols)
 {
   // build bit-wise objective function
 
-  prop_minimizet prop_minimize(boolbv);
-  prop_minimize.set_message_handler(get_message_handler());
+  prop_minimizet prop_minimize(boolbv, log.get_message_handler());
```

Comment:
Since you're doing this elsewhere: use `prop_minimize{...}` (applies several times in this PR).